12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,44 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+l */[a-z]*
```

Comment:
This works differently than was meant in the question, but nice nonetheless. 

The goal was to find files in the current directory that starts with a lowercase letter, while you found those files in subdirectories.